storage/migrations/02_oasis_3_runtimes.up.sql

```diff
@@ -98,6 +98,8 @@ CREATE TABLE oasis_3.runtime_events
   evm_log_params JSONB,
   related_accounts TEXT[]
 );
+CREATE INDEX ix_runtime_events_round ON oasis_3.runtime_events(runtime, round);  -- for sorting by round, when there are no filters applied
```

Comment:
would it be worthwhile to add tx_index to the end of this? for when we load a specific transaction's events